assignment-client/src/avatars/AvatarMixerSlavePool.cpp

```diff
@@ -75,10 +77,10 @@ void AvatarMixerSlavePool::broadcastAvatarData(ConstIter begin, ConstIter end,
                                                p_high_resolution_clock::time_point lastFrameTimestamp,
                                                float maxKbpsPerNode, float throttlingRatio) {
     _function = &AvatarMixerSlave::broadcastAvatarData;
-    _configure = [=](AvatarMixerSlave& slave) { 
+    _configure = [=](AvatarMixerSlave& slave) {
         slave.configureBroadcast(begin, end, lastFrameTimestamp, maxKbpsPerNode, throttlingRatio,
-            _priorityReservedFraction);
-   };
+                                    _priorityReservedFraction);
```

Comment:
indentation?

Reply:
I don't know whether I should copy/paste it again to have it aligned to the start of "begin" or force it to go back to where it was really.

Reply:
oh wait sorry this an argument not the body of a lambda, nvm me this looks correct (sorry reviewing on my phone)